docs/examples/frameworks/jax/model.py

```diff
@@ -0,0 +1,125 @@
+from functools import partial
```

Comment:
Here are basically the two scripts I've merged to try to have something as simple as possible :
* https://juliusruseckas.github.io/ml/flax-cifar10.html
* https://github.com/fattorib/Flax-ResNets/blob/master/main_flax.py

But I'm still blind as how to properly use jax. It would be much better if someone actually understanding how it works would bring comments but maybe later from an external contribution. For now, at least it installs and you can test that it trains

Reply:
I suggest we add these links in the docstring here

Reply:
Should be good now I think